app/scripts/ui/ZipcodeDialog.js

```diff
@@ -11,7 +11,7 @@ var ZipcodeDialog = React.createClass({
         return(
             <Modal title="Enter your zipcode" closeButton={false} onRequestHide={this.onHide}>
                 <div className="modal-body">
-                    <ZipcodeInput />
+                    <ZipcodeInput ackWhenValid={true} />
```

Comment:
Its the prop using which you can show/hide the 'Valid zipcode' message shown by zipcodeInputComponent. 

In the search page header, when we show zipcodeInput field, this message looks bad and not really required. So I added this prop as flag.